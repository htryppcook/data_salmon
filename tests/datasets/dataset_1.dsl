dataset_1 {
    int32 field0 = incremental_range(0,100,1)
    string field1 = value("abcd")
}