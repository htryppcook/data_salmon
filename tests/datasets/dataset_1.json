{
    "name": "dataset_1",
    "fields": [
        {
            "name": "field0",
            "type": "string",
            "strategy": "value",
            "arguments": ["abcd"]
        },
        {
            "name": "field1",
            "type": "int32",
            "strategy": "value",
            "arguments": [65535]
        }
    ]
}