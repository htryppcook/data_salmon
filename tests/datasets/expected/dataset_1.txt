0abcd1abcd2abcd3abcd4abcd5abcd6abcd7abcd8abcd9abcd